tests/buyProduct_test.js

```diff
@@ -22,43 +22,53 @@ let addressees = {
     city: "Kyiv",
 };
 
+const LinksGetter = require('../helpers/productLinksGetter.js');
+const home = require("../pages/home");
+let productLinks3 = LinksGetter.getLinks();
+console.log(productLinks3);
```

Comment:
remove. `console.log()` is used for debugging. If you want to keep it in test, add some text to identify what data is displayed in console. Applicable for all `console.log()`